packages/commons/src/config/fileManagerConfig.ts

```diff
@@ -13,6 +13,8 @@ const S3CustomServerConfig = z.preprocess(
         S3_CUSTOM_SERVER: z.literal("true"),
         S3_SERVER_HOST: z.string(),
         S3_SERVER_PORT: z.coerce.number().min(1001),
+        MINIO_ACCESS_KEY: z.string(),
+        MINIO_SECRET_KEY: z.string(),
```

Comment:
so far we used credentials in aws config file, if possible I would still follow the same approach

Reply:
Hi @galales, in this case, we cannot use, as far as I know, the aws config file, because we need the real credentials for KMS but the fake one for MINIO. So, if you agree with the approach of having S3 credentials I would delete completely the aws config local file.